src/app/components/attestation/attestation-create.jsx

```diff
@@ -135,159 +171,227 @@ function ChplAttestationCreate(props) {
+                  </Typography>
+                </CardContent>
+              </Card>
+              <br />
+              <Typography variant="body2">
+                For assistance with the Attestations submissions process, please visit the <Link>Health IT Feedback and Inquiry Portal</Link> to submit a ticket as applicable. For questions regarding the Attestations Condition and Maintenance of Certification requirement, please select the &quot;Attestations Condition&quot; category. For questions regarding a Condition and Maintenance of Certification requirement other than Attestations, please select the relevant Condition category. For technical assistance with this process, please select the &quot;CHPL&quot; category.
+              </Typography>
+            </Container>
           </>
         )}
-      { stage === 1
+      {stage === 1
         && (
           <>
-            <Typography variant="h2">
-              Attestations
-            </Typography>
-            {attestations
-              .map((attestation) => (
-                <div key={attestation.id}>
-                  <Typography variant="h3">
-                    { attestation.condition.name }
-                  </Typography>
-                  <FormControl key={attestation.id} component="fieldset">
-                    <FormLabel>{attestation.display}</FormLabel>
-                    <RadioGroup
-                      name={`response-${attestation.id}`}
-                      value={attestation.response.response}
-                      onChange={(event) => handleResponse(attestation, event.currentTarget.value)}
-                    >
-                      {attestation.validResponses
-                        .map((response) => (
-                          <FormControlLabel
-                            key={response.id}
-                            value={response.response}
-                            control={<Radio />}
-                            label={response.response}
-                          />
-                        ))}
-                    </RadioGroup>
-                  </FormControl>
-                </div>
-              ))}
+            <br />
+            <Container maxWidth="md">
+              <Typography gutterBottom variant="h2">
+                Attestations
+              </Typography>
+              <Card>
+                <CardContent>
+                  {attestations
+                    .map((attestation) => (
+                      <div key={attestation.id}>
+                        <Typography variant="subtitle1">
+                          {attestation.condition.name}
+                        </Typography>
+                        <FormControl key={attestation.id} component="fieldset">
+                          <FormLabel className={classes.nonCaps}>{attestation.display}</FormLabel>
```

Comment:
What would you think about lifting the `classes.nonCaps` an element up? To the `FormControl`? That way it would apply all the way down